transaction-builder/Cargo.toml

```diff
@@ -30,7 +30,7 @@ nimiq-transaction = { workspace = true }
 
 [dev-dependencies]
 hex = "0.4"
-rand = "0.8"
+rand = "0.9"
```

Comment:
Just noticed that some of the crates only have changes in the Cargo.toml.
Do they actually use the dependency at all?

Reply:
No, apparently not. Removed them.